src/components/taxonomicService/TaxonomicService.tsx

```diff
@@ -47,14 +48,23 @@ const TaxonomicService = () => {
             dispatch(setTaxonomicService(taxonomicService));
             dispatch(setIsApiOnline(true));
         },
-        ErrorHandler: () => {
-            setError(true);
-            dispatch(setIsApiOnline(false));
+        ErrorHandler: (error: Error) => {
+            setErrorMessage(error.message);
 
+            if (error?.cause !== 200) {
+                dispatch(setIsApiOnline(false));
+            }
         },
         params: { handle: taxonomicServiceID }
     });
 
+    /* Time out to check if the taxonomic service is still being loaded after 1.5 seconds */
+    setTimeout(() => {
+        if (fetch.loading) {
+            setDisplaySpinner(true);
+        };
+    }, 1500);
```

Comment:
1,5 sec feel like long before displaying a spinner but maybe it is not, is this a default strategy?

Reply:
Did not do any research into this. This is more so that the page does not flicker when loading.
This happens if you display the spinner instantly and if the request finishes very quickly you just see the spinner for a brief milliseconds, but it does mess with the lay-out of the page (it pushes it down).
Waiting for 1.5 second is so that it only displays the spinner, and changes the content, when the loading takes quite some time.
But I can change it to 1 second for example.